modules/changelogify_ai/src/Form/HumanizeItemForm.php

```
@@ -0,0 +1,254 @@
+      '#url' => Url::fromRoute('entity.changelogify_release.edit_form', [
+        'changelogify_release' => $changelogify_release->id(),
+      ]),
+      '#attributes' => ['class' => ['button']],
+    ];
+    return $form;
+  }
+
+  /**
+   * Requests a proposal without changing the release.
+   */
+  public function generateSubmit(array &$form, FormStateInterface $form_state): void {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$form'. (undefined)

(UnusedFormalParameter)